woocommerce/woocommerce
# Update to Playwright 1.32.3 and add UI command

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Updates Playwright to the latest version (1.32.3) and adds a command for running e2e tests in UI mode.  This PR replaces https://github.com/woocommerce/woocommerce/pull/37504 which had some merge difficulties.





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure e2e tests pass on CI
2. Pull branch
3. `pnpm env:start`
4. `pnpm test:e2e-pw-ui` -- ensure that Playwright starts in UI mode
5. `pnpm test:e2e-pw` -- ensure that tests pass
